src/Microsoft.Azure.WebJobs.Host/Blobs/Triggers/BlobTriggerBinding.cs

```diff
@@ -25,7 +25,6 @@ namespace Microsoft.Azure.WebJobs.Host.Blobs.Triggers
     internal class BlobTriggerBinding : ITriggerBinding
     {
         private readonly ParameterInfo _parameter;
-        private readonly IArgumentBinding<IStorageBlob> _argumentBinding;
```

Comment:
>IArgumentBinding [](start = 25, length = 16)

We no longer need the ArgumentBinding because the new BindToTrigger rule will bind the argument for us.